apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/service/AggregatesServiceDefault.java

```diff
@@ -247,42 +247,46 @@ private Uni<Aggregate> retrieveDataFromIpa(Aggregate aggregate) {
     }
 
     private Uni<Aggregate> retrieveCityCountyAndMapIpaFieldForUO(UOResource uoResource, Aggregate aggregateAppIo) {
-        return retrieveGeographicTaxonomies(uoResource.getCodiceComuneISTAT())
-                .onItem().transformToUni(geographicTaxonomyResource -> {
-                    mapIpaField(uoResource.getDescrizioneUo(), uoResource.getIndirizzo(), uoResource.getCap(), null, aggregateAppIo, geographicTaxonomyResource);
-                    return retrieveDigitalAddress(uoResource.getTipoMail1(), uoResource.getMail1(), uoResource.getCodiceFiscaleEnte(), aggregateAppIo);
-                });
+        return institutionApi.findInstitutionUsingGET(uoResource.getCodiceFiscaleEnte(), null, null)
+                .onItem().transformToUni(institutionResource ->
+                        retrieveGeographicTaxonomies(uoResource.getCodiceComuneISTAT())
```

Comment:
you could execute both call in parallel using `Uni.combine().all().unis` because they are independent from each other, for ex:

`Uni.combine().all().unis(institutionApi.findInstitutionUsingGET(uoResource.getCodiceFiscaleEnte(), null, null), retrieveGeographicTaxonomies(uoResource.getCodiceComuneISTAT()))`